src/applications/hca/tests/containers/ConfirmationPage.unit.spec.jsx

```diff
@@ -25,17 +25,14 @@ describe('hca <ConfirmationPage>', () => {
 
     const tree = SkinDeep.shallowRender(<ConfirmationPage form={form} />);
 
-    expect(tree.subTree('.confirmation-page-title').text()).to.contain(
```

Comment:
This entire test can be restored and a new test file can be set up for the ConfirmationPageNew component.

Reply:
Restored original test and created new one; commit pushed up.

Leaving this convo unresolved 'til the test passes below.

Reply:
All tests have now passed.